react-app/src/native/components/leagues/scheduleWeek.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react'
+import PropTypes from 'prop-types';
+import LeagueHelpers from '../../../utils/leaguehelpers';
+
+import { Text, Card, CardItem} from 'native-base';
+import { Table, Row } from 'react-native-table-component';
+import { StyleSheet } from 'react-native';
+import Loading from '../common/Loading';
+
+export default class ScheduleWeek extends React.Component {
+
+    constructor(props) {
+        super(props);
+    }
+
+    /**
+     * need to sort games by the location
+     * optimize some code as it is kinda slow when loading a league
+     * add locations names to schedule
+     * add bold to the times
+     * take care of unscheduled games
+     */
+
+    render() {
+
+        const { schedule, league } = this.props;
+
+        const flexArr = [4, 2, 2, 2];
+        const weekTable = {
+            header: ['Field', 'Dark', '', 'White'],
+            data: [],
+        }
+
+        //week1: { date: {}, matchTimes: [], matches: [{venue, time, team1, team2}]},
```

Comment:
If you want to display the game times the way you do you're going to need to update the schedule object so that each game time is it's own object containing the list of games happening at that time. Something like:

weeks: [
  week: {
    times: [
      {
        time: 6:30pm
        matches: [
          { teamOne, teamTwo, location }
        ]
      }
    ]
  }
]

that way you can use different styles on the time rows.